packages/build-utils/test/fixtures/25-multiple-lock-files-yarn/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "25-multiple-lock-files-yarn",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "mkdir -p public && (printf \"yarn version: \" && yarn -v) > public/index.txt"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "once": "^1.4.0"
```

Comment:
We can probably remove the dependency too?

Reply:
will lock files be valid without dependencies?

Reply:
I believe so but actually lets keep it since this is a real deployment